Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -709,46 +709,46 @@ class NewTabPageViewController: UIViewController {
         present(alert, animated: true, completion: nil)
     }
     
-    @objc private func longPressedBraveTodaySettingsButton() {
+    @objc private func longPressedBraveNewsSettingsButton() {
         assert(!AppConstants.buildChannel.isPublic,
                "Debug settings are not accessible on public builds")
-        let settings = BraveTodayDebugSettingsController(dataSource: feedDataSource)
+        let settings = BraveNewsDebugSettingsController(dataSource: feedDataSource)
         let container = UINavigationController(rootViewController: settings)
         present(container, animated: true)
     }
 }
 
 extension NewTabPageViewController: PreferencesObserver {
     func preferencesDidChange(for key: String) {
-        if !preventReloadOnBraveTodayEnabledChange {
+        if !preventReloadOnBraveNewsEnabledChange {
             collectionView.reloadData()
         }
-        if !isBraveTodayVisible {
+        if !isBraveNewsVisible {
             collectionView.verticalScrollIndicatorInsets = .zero
             feedOverlayView.headerView.alpha = 0.0
             backgroundButtonsView.alpha = 1.0
         }
-        preventReloadOnBraveTodayEnabledChange = false
+        preventReloadOnBraveNewsEnabledChange = false
     }
 }
 
 // MARK: - UIScrollViewDelegate
 extension NewTabPageViewController {
-    var isBraveTodayVisible: Bool {
+    var isBraveNewsVisible: Bool {
         #if NO_BRAVE_TODAY
```

Comment:
This could be renamed or removed if no longer used

Reply:
Renamed